playwright/pages/search.page.ts

```diff
@@ -87,6 +98,12 @@ export default class SearchPage {
       .locator("#select-topic")
       .getByText("Topic: Maps in education", { exact: true });
     this.nameFilter = this.page.getByRole("button", { name: "Name" });
+    this.nameOption = this.page
+      .locator("#select-name")
+      .getByText("Sheldonian Theatre", { exact: true });
+    this.nameSelected = this.page
+      .locator("#select-name")
+      .getByText("Name: Sheldonian Theatre", { exact: true });
```

Comment:
Sorry I know this has already been merged, but @jbdalton can you investigate in a follow-up PR if it's possible to use a more accessible selector here. Something like `this.page.getByRole("radio", { name: "Sheldonian Theater" })`? (haven't tested this myself but it seems like it could work)